recipes/goosefem/bld.bat

```diff
@@ -0,0 +1,13 @@
+
+cmake -G"NMake Makefiles" ^
```

Comment:
Can you use `NMake Makefiles JOM` or `Ninja` for multi-core compiling?

Reply:
@xhochy I'm kind of a newb in Windows, can you suggest how I should fix my `bld.bat`? (In the end I don't think that parallel build is very important here, as the only operation is the installation of a header-only library and the creation of some CMake files.)

Reply:
I normally use `Ninja` and `cmake --build . --target install --config Release` than I don't need to pass CPU Count in. I haven't seen that this is header-only, so this probably doesn't make any difference. Feel free to revert then as parallelisation is not necessary here, otherwise you need to add jom as a dependency.

Reply:
@xhochy Thanks! I have updated the PR with your suggestion. However, azure now seems to have a bug? (Appveyor seems already to have a bug)